vespaclient-container-plugin/src/main/java/com/yahoo/document/restapi/resource/DocumentV1ApiHandler.java

```diff
@@ -377,10 +377,6 @@ private DocumentOperationParameters parametersFromRequest(HttpRequest request, S
                                                           .map(parameters::withRoute)
                                                           .orElse(parameters);
                 break;
-            case CONDITION:
-                parameters = getProperty(request, ROUTE).map(parameters::withRoute)
-                                                        .orElse(parameters);
-                break;
```

Comment:
Oops, not intended to be here, but this is just removal of dead code that shouldn't be there in the first place.